Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -220,19 +228,112 @@ def search_records_by_report_soap_request(token, report_guid):
     return ET.tostring(root)
 
 
+def construct_generic_filter_condition(
+    condition_type: FilterConditionTypes,
+    operator: str,
+    field_name: str,
+    field_id: str,
+    search_value: str,
+    sub_elements_tags_values: dict[str, Any] | None = None
+) -> str:
+    """
+    Constructs an XML string representing any generic filter condition for searching records.
+
+    Args:
+        condition_type (FilterConditionTypes): The type of the filter condition.
+        operator (str): The comparison operator (e.g., 'Equals', 'GreaterThan').
+        field_name (str): The name of the application field.
+        field_id (str): The ID of the application field.
+        search_value (str): The value for the comparison.
+        sub_elements_tags_values (dict | None): Optional tag names and values to embed as sub-elements.
+
+    Returns:
+        str: An XML string representing the FilterCondition element.
+    """
+    root = ET.Element(condition_type.value)
+
+    operator_element = ET.SubElement(root, 'Operator')
+    operator_element.text = operator
+
+    field_element = ET.SubElement(root, 'Field', attrib={'name': field_name})
+    field_element.text = field_id
+
+    value_element = ET.SubElement(root, 'Value')
+    value_element.text = search_value
```

Comment:
```suggestion
    ET.SubElement(root, 'Value').text = search_value
```

Reply:
Fixed in commit cbeb616e474a7c5d67a4e15ac24b85286fcb58c8